contrib/ci/generate_docker.py

```
@@ -85,10 +85,11 @@ def get_container_cmd():
 content = content.replace("%%%DEPENDENCIES%%%", deps)
 
 # install android rust target
-rustup: str = ""
+rustup: list[str] = []
 if VARIANT == "android":
-    rustup = "RUN rustup default stable && rustup target add x86_64-linux-android"
-content = content.replace("%%%RUSTUP%%%", rustup)
+    rustup.append(f"COPY contrib/ci/android.sh .")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)